src/trunk.jl

```diff
@@ -27,6 +28,20 @@ The keyword arguments may include
 - `verbose::Int = 0`: if > 0, display iteration information every `verbose` iteration.
 - `verbose_subsolver::Int = 0`: if > 0, display iteration information every `verbose_subsolver` iteration of the subsolver.
 
+## Callback
+
+The callback is called after each iteration.
+The expected signature of callback is `(nlp, solver, wks)`, and the output of the callback is ignored.
```

Comment:
The fact that the user can actually stop the algorithm by setting `wks[:user_stop]` as `true` is slightly unclear.

Reply:
I made it more explicit.